regulations/views/comment.py

```diff
@@ -56,5 +52,21 @@ def preview_comment(request):
 def submit_comment(request):
     """Submit a comment to the task queue."""
     body = json.loads(request.body.decode('utf-8'))
-    tasks.submit_comment.delay(body)
-    return JsonResponse({'status': 'submitted'})
+    s3 = tasks.make_s3_client()
+    metadata_key = get_random_string(50)
+    metadata_url = s3.generate_presigned_url(
+        ClientMethod='get_object',
+        Params={
+            'Bucket': settings.ATTACHMENT_BUCKET,
+            'Key': metadata_key,
+        },
+    )
```

Comment:
Good point--the expiration defaults to an hour from this call, but can be set to expire at any timestamp. We can bump up this time, and/or make it configurable.